tools/vf-component-library/src/site/design-principles/index.njk

```diff
@@ -0,0 +1,140 @@
+---
+is_index: true
+title: Design Principles
+subtitle:
+date: 2021-02-02 13:29:50
+section: Design Principles
+order: 1
+tags:
+  - design principles
+  - sections
+layout: layouts/base.njk
+---
+
+{% include "navigation.njk" %}
+
+{% render '@vf-intro', {
+  "vf_intro_heading": title,
+  "vf_intro_subheading": "",
+  "vf_intro_lede": "",
+  "vf_intro_text": [
+    'Many organisations have design principles to guide their work. They are often described as ‘the star to sail your ship by’; a set of common beliefs and guidelines for a project, product, organisation, or initiative.'
+  ],
+  "vf_intro_badge": false
+} %}
+<section class="embl-grid">
+<div><!-- empty div --></div>
+<div class="vf-links vf-links__list--easy">
+  <h3 class="vf-links__heading">On this page</h3>
+  <ul class="vf-links__list | vf-list">
+    <li class="vf-list__item">
+      <a class="vf-list__link" href="#Sustainable">
+        Sustainable
+        <svg class="vf-icon vf-icon__arrow--down | vf-list__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M.249,7.207,11.233,19.678h0a1.066,1.066,0,0,0,1.539,0L23.751,7.207a.987.987,0,0,0-.107-1.414l-1.85-1.557a1.028,1.028,0,0,0-1.438.111L12.191,13.8a.25.25,0,0,1-.379,0L3.644,4.346A1.021,1.021,0,0,0,2.948,4a1,1,0,0,0-.741.238L.356,5.793A.988.988,0,0,0,0,6.478.978.978,0,0,0,.249,7.207Z"></path></svg>
```

Comment:
Curious why you didn't use the context or render here? Is it because the shortcoming of the vf-links-list?

Reply:
initial speed really, of getting something on the page.